Makefile

```diff
@@ -54,10 +54,10 @@ $(PLC_HOST_VARS): Makefile tcbsd-plc.yaml.template
 	envsubst < "tcbsd-plc.yaml.template" > "$@"
 
 run-bootstrap: $(PLC_HOST_VARS) tcbsd-bootstrap-playbook.yaml
-	ansible-playbook tcbsd-bootstrap-playbook.yaml
+	ansible-playbook tcbsd-bootstrap-playbook.yaml --extra-vars "target=vms_all ansible_ssh_private_key_file=${SSH_KEY_FILENAME}"
 
 run-provision: run-bootstrap tcbsd-provision-playbook.yaml
-	ansible-playbook tcbsd-provision-playbook.yaml
+	ansible-playbook tcbsd-provision-playbook.yaml --extra-vars "target=vms_all ansible_ssh_private_key_file=${SSH_KEY_FILENAME}"
```

Comment:
Arbitrarily noting this here:  It might be worth considering [ansible-vault](https://docs.ansible.com/ansible/2.9/user_guide/vault.html) to manage secrets.  But that's just an association I made, I'm not convinced yet that it's the right solution